interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName[:len(baseName)-3]+".bpf")
```

Comment:
I *thought* that snapd was particular about what was in `dirs.SnapSeccompDir` and it would remove things it didn't know about whenever profile regeneration happened. Has this changed?